sourced/ml/transformers/basic.py

```diff
@@ -123,6 +124,7 @@ def __init__(self, fields: Union[list, tuple], **kwargs):
     def __call__(self, df: DataFrame) -> DataFrame:
         res = df.select(self.fields)
         if self.explained:
+            self._log.info("toDebugString():\n%s", res.explain(True).decode())
```

Comment:
I think `res.explain()` prints to stdout, also is very impresise and loses the previous pipeline. Should be avoided.